config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get "/users/:id", to: "users#show"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
   # Defines the root path route ("/")
```

Comment:
- [ ] **Optional:** It is conventional always to have a root part so that the app loads into some page with content. Currently, the load is empty until I navigate to `/users`. Please keep this in mind for future projects.

Reply:
Thank you very much for making me aware of it. I'll surely add it in future projects as well as in my current project.